src/main/java/uk/gov/pay/products/client/publicapi/PaymentRequest.java

```diff
@@ -1,10 +1,12 @@
 package uk.gov.pay.products.client.publicapi;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
We shouldn’t need this here: we never deserialise anything into a `PaymentRequest` object, so we don’t have to worry about unknown properties